proposed/http-client/http-client.md

```diff
@@ -43,10 +43,9 @@ interface HttpClient
     /**
      * Sends a PSR-7 request.
      *
-     * If a request is sent without any prior configuration, an exception MUST NOT be thrown
-     * when a response is recieved, no matter the HTTP status code.
+     * An exception MUST NOT be thrown when a response is recieved, no matter the HTTP status code.
```

Comment:
Yes, correct. We should not throw an exception in **that** case. That is still a valid version number. It is not officially recognised but it is valid. 

However, if the server responds with something that cannot be parsed as HTTP we should throw an exception.